src/Visitors/RemoveArrayPropertyItem.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace RonasIT\Larabuilder\Visitors;
+
+use Illuminate\Support\Arr;
+use PhpParser\Node;
+use PhpParser\Node\Expr\Array_;
+use PhpParser\Node\Expr\ConstFetch;
+use PhpParser\Node\Scalar;
+use PhpParser\Node\Stmt\Property;
+use RonasIT\Larabuilder\Exceptions\UnexpectedPropertyTypeException;
+
+class RemoveArrayPropertyItem extends SetPropertyValue
+{
+    protected array $valuesToRemove;
+
+    public function __construct(
+        protected string $name,
+        array $valuesToRemove,
+    ) {
+        $this->valuesToRemove = array_map(
+            callback: fn ($value) => $this->getPropertyValue($value)[0],
+            array: $valuesToRemove,
+        );
+
+        parent::__construct($name, []);
+    }
+
+    /** @param Property $node */
+    protected function updateNode(Node $node): void
+    {
+        $arrayProperty = $node->props[0]->default;
+
+        if (!$arrayProperty instanceof Array_) {
+            throw new UnexpectedPropertyTypeException(
+                property: $this->name,
+                expectedType: 'array',
+                actualType: (is_null($node->type)) ? 'null' : (string) $node->type,
```

Comment:
it would be better to allow to receive null in exception

```suggestion
                actualType: $node->type,
```